R/est.incidence.by.R

```diff
@@ -101,6 +101,27 @@ est.incidence.by <- function(
     return(to_return)
   }
 
+  if (!any(is.element(strata,pop_data %>% names()))) {
+    # search strata variable from pop_data
+    strata_var <-
+      grep(
+        x = pop_data %>% names(),
+        value = TRUE,
+        pattern = strata,
+        ignore.case = TRUE
+      )
+
+    cli::cli_abort(
+      class = "missing_var",
+      message = c(
+        "Can't find the column {.var {strata}} in {.envvar pop_data}.",
```

Comment:
please revise this inline markup; `pop_data` is a [function argument](https://cli.r-lib.org/reference/inline-markup.html#:~:text=arg%20for%20a%20function%20argument), not an environment variable (see https://cli.r-lib.org/reference/inline-markup.html)